bgpd/bgpd.c

```diff
@@ -4237,6 +4237,14 @@ int bgp_delete(struct bgp *bgp)
 		}
 	}
 
+	// Clean BGP address family parameters
```

Comment:
Please drop this comment or replace with `/* */`.

Reply:
Done